kolibri/plugins/management/assets/src/device_management/state/actions/managePermissionsActions.js

```diff
@@ -0,0 +1,116 @@
+import {
+  DevicePermissionsResource,
+  NewDevicePermissionsResource,
+  FacilityUserResource,
+} from 'kolibri.resources';
+import { handleApiError } from 'kolibri.coreVue.vuex.actions';
+import groupBy from 'lodash/groupBy';
+import mapValues from 'lodash/mapValues';
+import head from 'lodash/head';
+import isEmpty from 'lodash/isEmpty';
+
+function fetchFacilityUsers() {
+  return FacilityUserResource.getCollection().fetch();
+}
+
+function fetchDevicePermissions() {
+  return DevicePermissionsResource.getCollection()
+    .fetch({}, true)
+    ._promise.then(function transform(permissions) {
```

Comment:
Stylistic: prefer ES6 style functions, especially for anonymous functions like this.